advocacy_docs/edb-postgres-ai/ai-ml/overview.mdx

```diff
@@ -7,7 +7,7 @@ prevNext: True
 
 At the heart of EDB Postgres® AI is the EDB Postgres AI database (pgai). This builds on Postgres's flexibility and extends its capability to include storing the vector data of embeddings.
 
-The pgai extension is currently available as a tech preview. It will be continuously extended with new functions. Here comes a description of the major functionality available to date.
+The pgai extension is currently available as a tech preview. It will be continuously extended with new functions. Here is a description of the major functionality available to date.
```

Comment:
Just a suggestion: 

```suggestion
The pgai extension is currently available as a tech preview. It will be continuously extended with new functions. This overview presents the functionality available to date.
```